src/loader.ts

```diff
@@ -116,14 +116,14 @@ export default function loader(
     //
     // This way, the only modified file has to be the _app.js.
     if (hasGetInitialPropsOnAppJs) {
-      return pageNoExt === '/_app'
+      return pageNoExt.startsWith('/_app')
```

Comment:
the problem is that this is not going to work in monorepos...

Reply:
well, maybe yes is fine. If before was just `===` and was working...  Is `pageNoExt` only the page and not all the path... So then yes.  This is to fix these cases `/_app.page.ts`, right?

Reply:
Yes, this is a fix for `_app.page.tsx` - custom `pageExtensions`.